weave

```diff
@@ -1211,6 +1212,7 @@ launch_proxy() {
         --entrypoint=/home/weave/weaveproxy \
         $WEAVEPROXY_DOCKER_ARGS $EXEC_IMAGE $COVERAGE_ARGS $PROXY_ARGS)
     wait_for_log $PROXY_CONTAINER_NAME "proxy listening"
+    ln -sf -T /var/run/weave/weave.sock /var/run/weave.sock
```

Comment:
which makes me wonder... we could use a little trick here so we don't need the `/var/run` mount in `exec_remote` even now... run the `ls` in a container (yet another incarnation of the `weavexec` image) that mounts `/var/run/`.